web/docs/tql2/functions.md

```diff
@@ -211,7 +211,7 @@ Function | Description | Example
 :--------|:-------------|:-------
 [`int`](functions/int.md) | Casts an expression to a signed integer | `int(-4.2)`
 [`uint`](functions/uint.md) | Casts an expression to an unsigned integer | `uint(4.2)`
-[`float`](functions/float.md) | Casts an expression to a float | `float(42)`
+[`double`](functions/double.md) | Casts an expression to a double | `double(42)`
```

Comment:
We should make a decision regarding the naming (`double` vs `float`) here soon-ish. I'm leaving this conversation here as a reminder to not merge before.